SQL/0000-00-00-schema.sql

```diff
@@ -982,8 +982,9 @@ INSERT INTO `permissions` VALUES
     (11,'data_entry','Data entry','1'),
     (12,'examiner_view','Add and certify examiners','2'),
     (13,'examiner_multisite','Across all sites add and certify examiners','2'),
-    (14,'timepoint_flag','Edit exclusion flags','2'),
-    (15,'timepoint_flag_evaluate','Evaluate overall exclusionary criteria for the timepoint','2'),
+    (14,'training','View and complete training','2'),
```

Comment:
But no old projects will be affected by this, as it is only in the default schema, so only for new installs.